platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

```diff
@@ -151,6 +151,11 @@ However, if the build is configured to fail on warnings during Kotlin script or
 
 A standard Gradle deprecation warning will be printed upon use when these methods are fully deprecated in a future version.
 
+==== Removed incubating method `ObjectFactory#dependencyCollector()`
```

Comment:
Is there a reason this is in the section "Upgrading from 8.12 and earlier" ?

Reply:
This is an old PR I just revived. Moved to 8.14